ros_opt_data_noetic_core22/snap/snapcraft.yaml

```diff
@@ -0,0 +1,23 @@
+name: ros-opt-data-example
+version: '0.1'
+summary: Snap using ros-opt-data interface
+description: |
+    Example on how to enable access to the host folder /opt/ros from inside a snap
+base: core22
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict
+
+architectures:
+  - build-on: [amd64]
+    build-for: [amd64]
+
+apps:
+  run-example-cat:
+    command: ./cat_files_content.sh
+    plugs: [ros-opt-data]
```

Comment:
Is it expected that your interface is not documented? https://snapcraft.io/docs/supported-interfaces

Is it autoconnect?

Reply:
https://forum.snapcraft.io/t/the-ros-opt-data-interface/39981 now it's documented, and yes, it's autoconnect